lib/instrumentation/http.test.js

```diff
@@ -142,7 +142,7 @@ test("node 10.9.0+ url + options", (done) => {
   tracker.setTracked(newMockContext());
 
   http.get(
-    "http://localhost:80",
+    "http://localhost:9009",
```

Comment:
I _think_ they're trying to test that the port in `options` args will replace the port in the `url` here, see their ; might want to change it back? See [their lines 37-38ish](https://github.com/notmatt/beeline-nodejs/pull/1/files#diff-a04e82062dd81ac97fa3e7865c8b163aec88621d85032ceadcc142841db4c02aL37-L38).